content/documentation/reference-guide.md

```diff
@@ -22,11 +27,6 @@ parent = "Documentation"
 * API documentation: [JavaDoc](/documentation/1.4/api/)
 * [Migration notes](https://github.com/mapstruct/mapstruct/releases/tag/1.4.2.Final)
 
-**Spring Extensions** (February 15th 2021; latest milestone release)
```

Comment:
I think that it is better to keep the extension release just below the last mapstruct core one

Reply:
I figured you might say that. :laughing:

Reply:
I think that it can be a bit confusing otherwise. Thanks for the change. I merged this